setup.py

```diff
@@ -74,6 +77,19 @@ def clean_version(v):
       setup_requires=[],
       )
 
+if current_platform != "windows":
+    bash_file = "~/.bash_profile" if current_platform == "darwin" else "~/.bashrc"
+    argcomplete_command = 'eval "$(register-python-argcomplete retriever)"'
+    with open(os.path.expanduser(bash_file), "a+") as bashrc:
+        bashrc.seek(0)
+        # register retriever for arg-completion if not already registered
+        # whenever a new shell is spawned
+        if argcomplete_command not in bashrc.read():
+            bashrc.write(argcomplete_command + "\n")
+            bashrc.close()
+    os.system("activate-global-python-argcomplete")
```

Comment:
Is this the only command needed for windows `os.system("activate-global-python-argcomplete")` basing on the if statement `if current_platform != "windows":`